app/src/main/java/com/example/chickenmasala/data/interactors/GuessGamesInteractor.kt

```diff
@@ -13,52 +13,58 @@ class GuessGamesInteractor(private val dataSource: FoodDataSource<RecipeEntity>)
             GuessGamesName.GUESS_THE_EXSTING_INGREDIENT -> guessExistingIngredient()
             GuessGamesName.GUESS_THE_MEAL -> guessMeal()
         }
-
     }
 
-
-    private fun guessMeal(): QuestionGames {
+    private fun guessExistingIngredient(): QuestionGames {
         val correctName = randomRecipe.name
         val correctIngredent = randomRecipe.cleanedIngredients.random()
-        val result = dataSource.getAllItems()
+        val wrongAnswersRecipe = dataSource.getAllItems()
```

Comment:
to make it accessible from multiple locations, you could declare this variable within the class's scope.

Reply:
i tried this but i got fixed list of wrong choices for each question just the correct answers are  changed so i made it local for each function